samples/meetings-details-tab/nodejs/server/api/homeController.js

```diff
@@ -0,0 +1,40 @@
+const configuration = require('dotenv').config();
+const store = require('../services/store')
+const {createAdaptiveCard }= require('../services/AdaptiveCardService')
+const env = configuration.parsed;
+const appID = env.BotId;
```

Comment:
Please remove `appId` and `appPassword variable`, You can directly use env.BotId and env.BotPassword in place